src/config/index.js

```diff
@@ -124,7 +124,7 @@ export const config = convict({
     enabled: {
       doc: 'Enable Redis on your Frontend.',
       format: Boolean,
-      default: true,
+      default: false,
```

Comment:
I think redis will be needed as I am using the hapi/yar plugin. See the comment in the /src/server/common/helpers/session-cache/cache-engine